### ISSUE TITLE
[5.0] Dark mode - All blue text fails WCAG colour contrast requirements

### ISSUE THREAD

**Issue by C-Lodder** [joomla/joomla-cms#41969]
### Steps to reproduce the issue

Go to your Joomla backend and ensure dark mode is enabled.

### Actual result

All blue text (mainly links) fail the minimum required colour contrast.

Example:

![image](https://github.com/joomla/joomla-cms/assets/2019801/dde60070-5db1-4160-b495-930a513d8956)

**Comment by brianteeman** [comment]
I assume thats why @wilsonge create #41925 ??

**Comment by wilsonge** [comment]
> I assume thats why @wilsonge create https://github.com/joomla/joomla-cms/pull/41925 ??

Yup that was the intended fix for links. I'll keep working through the a11y stuff as people report it otherwise.

**Comment by brianteeman** [comment]
@wilsonge its hard to really do that when the colours are supposed to be changed. so it will have to be checked again afterwards making any checks now a waste of time

**Comment by brianteeman** [comment]
No longer an issues with the rewritten dark mode and can be closed

![image](https://github.com/joomla/joomla-cms/assets/1296369/67569bfd-b9df-4c49-b589-39a244fb4228)

**Comment by Quy** [comment]
Thanks Brian.
